src/components/site/SummaryList.md

```diff
@@ -0,0 +1,104 @@
+```jsx
+const publicWorksSummaryItems = [
+  {
+    id: 1,
+    icon: "fas fa-map-marker-alt",
+    heading: "Main Office",
+    content:
+      "<p> 111 West Chesapeake Avenue<br /> Suite 307<br /> Towson, Maryland 21204</p>"
+  },
+
+  {
+    id: 2,
+    icon: "far fa-clock",
+    heading: "Hours:",
+    content: "<p>Monday through Friday<br />8 a.m. to 4:30 p.m.</p>"
+  },
+  {
+    id: 3,
+    icon: "far fa-envelope",
+    heading: "Email:",
+    content: "<p><a href=`#`>publicworks@​baltimorecountymd.gov</a></p>"
+  },
+  {
+    id: 4,
+    icon: "fas fa-mobile-alt",
+    heading: "Phone:",
+    content:
+      "<p><a href=`tel:410-887-6789`>410-887-6789</a></p><p><h3>Emergency Dispatch:</h3></p><p><a href=`tel:410-887-6789`>410-887-6789</a></p>"
+  },
+  {
+    id: 5,
+    icon: "fas fa-exclamation-circle",
+    heading: "BALTCOGO311",
+    content: " <p><a href=`#`>Report a problem</a> online.</p>"
+  },
+  {
+    id: 6,
+    icon: "fas fa-user",
+    heading: "Director:",
+    content: "<p>Steve Walsh, P.E.</p>"
+  }
+];
+
+<SummaryList items={publicWorksSummaryItems} />;
+```
+
+Html Snippet:
+
+```html
+
+<div>
```

Comment:
this is probably going to need update, and ilooks like it could use some formatting.

Reply:
updated